functions/New-DbaAgentOperator.ps1

```diff
@@ -0,0 +1,311 @@
+        The default is NotifyEmail.
+
+    .PARAMETER Force
+        If this switch is enabled, the Operator will be dropped and recreated on Destination.
+
+    .PARAMETER EnableException
+        By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+        This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags: Agent, Operator
+        Author: Tracy Boggiano (@TracyBoggiano), databasesuperhero.com
+
+        Website: https://dbatools.io
+        Copyright: (c) 2018 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+    .LINK
+        https://dbatools.io/New-DbaAgentOperator
+
+    .EXAMPLE
+        PS:\> New-DbaAgentOperator -SqlInstance sql01 -Operator DBA -OperatorEmail operator@operator.com -PagerDay Everyday
+
+        This sets a new operator named DBA with the above email address with default values to alerts everyday
+        for all hours of the day.
+
+    #>
+    [CmdletBinding(DefaultParameterSetName = "Default", SupportsShouldProcess, ConfirmImpact = "Medium")]
+    param (
+        [parameter(Mandatory)]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [parameter(Mandatory)]
+        [string]$Operator,
+        [string]$EmailAddress,
+        [string]$NetSendAddress,
+        [string]$PagerAddress,
+        [ValidateSet('EveryDay', 'Weekdays', 'Weekend', 'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday')]
+        [string]$PagerDay,
+        [string]$SaturdayStartTime,
+        [string]$SaturdayEndTime,
+        [string]$SundayStartTime,
+        [string]$SundayEndTime,
+        [string]$WeekendStartTime,
+        [string]$WeekendEndTime,
+        [switch]$IsFailsafeOperator = $false,
+        [string]$FailsafeNotificationMethod = "NotifyEmail",
+        [switch]$Force = $false,
+        [switch]$EnableException
+    )
+
+    begin {
+        if ($Force) { $ConfirmPreference = 'none' }
+    }
+
+    process {
+        if ($null -eq $EmailAddress -and $null -eq $NetSendAddress -and $null -eq $PagerAddress) {
+            Stop-Function -Message "You must specify either an EmailAddress, NetSendAddress, or a PagerAddress to be able to create an operator."
+            return
+        }
+        foreach ($instance in $SqlInstance) {
+            try {
+                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $serverSqlCredential
+            } catch {
+                Stop-Function -Message "Error occurred while establishing connection to $instance" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
```

Comment:
Please change the message to just be `Failed.`

We've adjusted our message system to include the instance now, but requires the message only contain `Failed.` to trigger it.